syntax = "proto3";
package basic;




import "tool_sdk/model/tool/tool.proto";

/*
BatchGetToolDetail请求
*/
message BatchGetToolDetailRequest {
    /*
    工具id，用逗号隔开，'.'后为指定版本Id，不指定则不填 例：id1.vId1,id2,id3.vId3
    */
    string toolIds = 1 ;
}

/*
BatchGetToolDetail返回
*/
message BatchGetToolDetailResponse {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    错误信息
    */
    string error = 2 ;
    /*
    返回消息
    */
    string message = 3 ;
    /*
    工具信息列表
    */
    repeated tool.Tool data = 4 ;
}

/*
BatchGetToolDetailApi返回
*/
message BatchGetToolDetailResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    basic.BatchGetToolDetailResponse data = 4 ;
}

